File: Tabel-Aritmetika.cpp
#include <iostream>

float tabel_penjumlahan ();
float tabel_pengurangan ();
float tabel_perkalian ();
float tabel_pembagian ();

int main ()
{
	unsigned short pilihan = 0;
	
    do
    {
        std::cout << "========== Pembuat Tabel Aritmetika ==========\n";
        std::cout << "1. Penjumlahan\n";
        std::cout << "2. Pengurangan\n";
        std::cout << "3. Perkalian\n";
        std::cout << "4. Pembagian\n";
        std::cout << "5. Keluar\n";
        std::cout << "Pilih salah satu: ";
        std::cin >> pilihan;
        
        switch (pilihan)
        {
            case 1:
                tabel_penjumlahan ();
                break;
            case 2:
                tabel_pengurangan ();
                break;
            case 3:
                tabel_perkalian ();
                break;
            case 4:
                tabel_pembagian ();
                break;
            case 5:
                std::cout << "Terima kasih telah menggunakan program ini";
            default:
                std::cout << "Input error\n";
                break;
        }
    } while (pilihan > 5 || pilihan < 0);
    
     
     return 0;
}

float tabel_penjumlahan ()
{
	float batas = 0, bilangan = 0;
	
	std::cout << "Masukkan bilangan: ";
    std::cin >> bilangan;
    std::cout << "Masukkan batas hitung: ";
    std::cin >> batas;

    for (int i = 1; i <= batas; i++)
    {
        std::cout << bilangan << " + " << i << " = " << bilangan + i << "\n";
    }
}

float tabel_pengurangan ()
{
	float batas = 0, bilangan = 0;
	
	std::cout << "Masukkan bilangan: ";
    std::cin >> bilangan;
    std::cout << "Masukkan batas hitung: ";
    std::cin >> batas;

    for (int i = 1; i <= batas; i++)
    {
        std::cout << bilangan << " - " << i << " = " << bilangan - i << "\n";
    }
}

float tabel_perkalian ()
{
	float batas = 0, bilangan = 0;
	
	std::cout << "Masukkan bilangan: ";
    std::cin >> bilangan;
    std::cout << "Masukkan batas hitung: ";
    std::cin >> batas;

    for (int i = 1; i <= batas; i++)
    {
        std::cout << bilangan << " x " << i << " = " << bilangan * i << "\n";
    }
}

float tabel_pembagian ()
{
	float batas = 0, bilangan = 0;
	
	std::cout << "Masukkan bilangan: ";
    std::cin >> bilangan;
    std::cout << "Masukkan batas hitung: ";
    std::cin >> batas;

    for (int i = 1; i <= batas; i++)
    {
        std::cout << bilangan << " : " << i << " = " << bilangan / i << "\n";
    }
}

File: Tabel Aritmetika/TabelAritmetika.java
import java.util.Scanner;

public class TabelAritmetika
{
    Scanner scanner = new Scanner (System.in);

    void tabel_penjumlahan ()
    {
        float batas = 0, bilangan = 0;

        System.out.print ("Masukkan bilangan: ");
        bilangan = scanner.nextFloat ();

        System.out.print ("Masukkan batas hitung: ");
        batas = scanner.nextFloat ();

        for (int i = 1; i <= batas; i++)
        {
            System.out.printf ("%.2f + %d = %.2f\n", bilangan, i, bilangan + i);
        }
    }

    void tabel_pengurangan ()
    {
        float batas = 0, bilangan = 0;

        System.out.print ("Masukkan bilangan: ");
        bilangan = scanner.nextFloat ();

        System.out.print ("Masukkan batas hitung: ");
        batas = scanner.nextFloat ();

        for (int i = 1; i <= batas; i++) {
            System.out.printf ("%.2f - %d = %.2f\n", bilangan, i, bilangan - i);
        }
    }

    void tabel_perkalian() {
        float batas = 0, bilangan = 0;

        System.out.print ("Masukkan bilangan: ");
        bilangan = scanner.nextFloat ();

        System.out.print ("Masukkan batas hitung: ");
        batas = scanner.nextFloat ();

        for (int i = 1; i <= batas; i++)
        {
            System.out.printf ("%.2f x %d = %.2f\n", bilangan, i, bilangan * i);
        }
    }

    void tabel_pembagian ()
    {
        float batas = 0, bilangan = 0;

        System.out.print ("Masukkan bilangan: ");
        bilangan = scanner.nextFloat ();

        System.out.print ("Masukkan batas hitung: ");
        batas = scanner.nextFloat ();

        for (int i = 1; i <= batas; i++)
        {
            System.out.printf ("%.2f : %d = %.2f\n", bilangan, i, bilangan / i);
        }
    }

    public static void main (String [] args)
    {
        TabelAritmetika ta = new TabelAritmetika ();    

        byte pilihan = 0;

        do {
            System.out.println("========== Pembuat Tabel Aritmetika ==========");
            System.out.println("1. Penjumlahan");
            System.out.println("2. Pengurangan");
            System.out.println("3. Perkalian");
            System.out.println("4. Pembagian");
            System.out.println("5. Keluar");
            System.out.print("Pilih salah satu: ");
            pilihan = ta.scanner.nextByte ();

            switch (pilihan) {
                case 1:
                    ta.tabel_penjumlahan ();
                    break;
                case 2:
                    ta.tabel_pengurangan ();
                    break;
                case 3:
                    ta.tabel_perkalian ();
                    break;
                case 4:
                    ta.tabel_pembagian ();
                    break;
                case 5:
                    System.out.println ("Terima kasih telah menggunakan program ini");
                    break;
                default:
                    System.out.println ("Input error");
                    break;
            }
        } while (pilihan > 5 || pilihan < 0);
    }
}